django/db/migrations/operations/fields.py

```diff
@@ -354,6 +363,9 @@ def database_backwards(self, app_label, schema_editor, from_state, to_state):
     def describe(self):
         return "Rename field %s on %s to %s" % (self.old_name, self.model_name, self.new_name)
 
+    def suggest_migration_name(self):
+        return "rename_%s_%s_%s" % (self.model_name_lower, self.old_name_lower, self.new_name_lower)
```

Comment:
I think it makes sense to use the same order like in `describe()`.

Reply:
I agree